.github/workflows/tests.yml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: cargo llvm-cov --workspace --features unstable --no-fail-fast --ignore-filename-regex "aws-lc-(fips-)?sys/.*" --lcov --output-path ${{ runner.temp }}/lcov.info
       - name: Run FIPS coverage
         working-directory: ./aws-lc-rs
-        run: cargo llvm-cov --workspace --features unstable,fips --no-fail-fast --ignore-filename-regex "aws-lc-(fips-)?sys/.*" --lcov --output-path ${{ runner.temp }}/lcov-fips.info
+        run: cargo llvm-cov --workspace --features unstable --no-fail-fast --ignore-filename-regex "aws-lc-(fips-)?sys/.*" --lcov --output-path ${{ runner.temp }}/lcov-fips.info
```

Comment:
Accident? W/o the "fips" feature this looks the same as previous CI step.

Reply:
Oh, I think I had added "fips" to one line in an earlier revision or something, then removed it a little to aggressively I guess.